inst/safetyGraphics_app/modules/renderSettings/renderSettings.R

```diff
@@ -45,12 +45,9 @@ source("modules/renderSettings/util/updateSettingStatus.R")
 #' \item{"settings"}{Upadted settings object based on UI/user selections}
 #' \item{"status"}{Result from validateSettings() for originally selected data + updated settings object}
 #'
-renderSettings <- function(input, output, session, data, settings, status){
+renderSettings <- function(input, output, session, data, settings, status, metadata, charts){
```

Comment:
Probably worth updating the documentation here with the new inputs. Also, not entirely clear if we are using the new `charts` option in the function.

Reply:
@jwildfire good catch. I can work on this.